sdk/tables/azure-data-tables/README.md

```diff
@@ -191,9 +201,25 @@ table_client = TableClient.from_connection_string(conn_str="<connection_string>"
 entity = table_client.query_entities(filter=my_filter)
 ```
 
+## Troubleshooting
+When you interact with the Azure table library using the Python SDK, errors returned by the service respond ot the same HTTP status codes for [REST API](tables_rest) requests.
+
+For examples, if you try to create a table that already exists, a `409` error is returned indicating "Conflict".
+```python
+service_client = TableServiceClient(connection_string)
+
+# Create the table if it does not already exist
+tc = service_client.create_table_if_not_exists(table_name)
+
+try:
+    service_client.create_table(table_name)
+except HttpResponseError:
+    print("Table with name {} already exists".format(table_name))
+```
+
 ## Optional Configuration
 
```

Comment:
This looks like it might go under "Key Concepts". As a general rule, the main headings should stick as close to the guidelines as possible: https://review.docs.microsoft.com/en-us/help/contribute-ref/contribute-ref-how-to-document-sdk?branch=master#readme